Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1642,10 +1642,23 @@ Limitations
       which uses eBPF cgroup hooks to implement the service translation. Using it with libceph
       deployments currently requires support for the getpeername(2) hook address translation in
       eBPF, which is only available for kernels v5.8 and higher.
-    * In order to support nfs in the kernel with the socket-LB feature, ensure that
-      kernel commit ``0bdf399342c5 ("net: Avoid address overwrite in kernel_connect")``
-      is part of your underlying kernel. Linux kernels v6.6 and higher support it. Older
-      stable kernels are TBD. For a more detailed discussion see :gh-issue:`21541`.
+    * NFS and SMB mounts may break when mounted to a ``Service`` cluster IP while using socket-LB,
+      This issue is known to impact Longhorn, Portworx, and Robin, but may impact other storage
+      system that implement ``ReadWriteMany`` volumes using this pattern. To avoid this problem,
```

Comment:
Grammar nit: `system` -> `systems`.

```suggestion
      systems that implement ``ReadWriteMany`` volumes using this pattern. To avoid this problem,
```

Reply:
Done